File: mobile/Keep_Notes_Android.java
package mobile;

import io.appium.java_client.android.AndroidDriver;
import java.net.URL;
import org.junit.After;
import org.junit.Before;
import org.junit.Test;
import org.openqa.selenium.remote.DesiredCapabilities;


public class Keep_Notes_Android {

    private static final String APPIUM = "http://localhost:4723/wd/hub";
    private AndroidDriver driver;


    @Before
    public void setUp() throws Exception {
        DesiredCapabilities caps = new DesiredCapabilities();
        caps.setCapability("platformName", "Android");
        caps.setCapability("platformVersion", "13");
        caps.setCapability("deviceName", "Android Emulator");
        caps.setCapability("automationName", "UiAutomator2");
        this.driver = new AndroidDriver(new URL("http://localhost:4723/wd/hub"), caps);
    }

    @After
    public void tearDown() {
        if (this.driver != null) {
            this.driver.quit();
        }

    }

    @Test
    public void test() {
        System.out.println("test!");
    }
}

File: mobile/Keep_Notes_Android.py
from appium import webdriver
from selenium.webdriver.common.by import By
from applitools.selenium import Eyes
import time

desired_capabilities = {
    "platformName": "Android",
    "platformVersion": "9.0",
    "deviceName": "Pixel 2",
    # "automationName": "Appium",
    # "app": 	'//android.widget.TextView[@content-desc="Keep Notes"]'

}
driver = webdriver.Remote('http://127.0.0.1:4723/wd/hub', desired_capabilities)
# Open app
driver.find_element(By.XPATH,'//android.widget.TextView[@content-desc="Keep Notes"]').click()
# driver.find_element(By.ID,'Keep Notes').click()
time.sleep(2)
# Creating note
# driver.find_element(By.ID,'com.google.android.keep:id/new_note_button').click()
# time.sleep(5)
# title = driver.find_element(By.ID, 'com.google.android.keep:id/editable_title')
# title.send_keys("Test title")
# body = driver.find_element(By.ID, 'com.google.android.keep:id/edit_note_text')
# body.send_keys('Test body')